From 2bd6a22c67eb197d49fcddd8fb4f1ae33de8543b Mon Sep 17 00:00:00 2001
From: Davor Joja <davorjoja@logicbricks.com>
Date: Tue, 27 Oct 2015 17:11:00 +0100
Subject: [PATCH 601/793] gpu: drm: xylon: Added support for logiCVC v5.0

Added compatible property for supporting logiCVC v5.0
from driver devicetree entry.
Removing 4.01.a compatible string which was incorrectly there.

Signed-off-by: Davor Joja <davorjoja@logicbricks.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/gpu/drm/xylon/xylon_logicvc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/xylon/xylon_logicvc.c b/drivers/gpu/drm/xylon/xylon_logicvc.c
index 1cf18e0..74e1699 100644
--- a/drivers/gpu/drm/xylon/xylon_logicvc.c
+++ b/drivers/gpu/drm/xylon/xylon_logicvc.c
@@ -900,7 +900,7 @@ void xylon_cvc_get_fix_parameters(struct xylon_cvc *cvc,
 
 static const struct of_device_id cvc_of_match[] = {
 	{ .compatible = "xylon,logicvc-4.00.a" },
-	{ .compatible = "xylon,logicvc-4.01.a" },
+	{ .compatible = "xylon,logicvc-5.0" },
 	{/* end of table */}
 };
 
-- 
1.7.5.4

